conf/config.json

```diff
@@ -13,6 +13,7 @@
       "timeout": "0.0",
       "password": ""
      },
+    "lost_password_link" => "https://$domain$path/sso/password.html",
```

Comment:
```suggestion
    "lost_password_link": "https://$domain$path/sso/password.html",
```

Reply:
How does it work for users without sso account (only registered with Nextcloud)?